packages/lib/constants.ts

```diff
@@ -152,7 +152,8 @@ export const BOOKER_NUMBER_OF_DAYS_TO_LOAD = parseInt(
 export const CLOUDFLARE_SITE_ID = process.env.NEXT_PUBLIC_CLOUDFLARE_SITEKEY;
 export const MINIMUM_NUMBER_OF_ORG_SEATS = 30;
 export const ORG_SELF_SERVE_ENABLED = process.env.NEXT_PUBLIC_ORG_SELF_SERVE_ENABLED === "1";
-export const ORG_MINIMUM_PUBLISHED_TEAMS_SELF_SERVE = 0;
+export const ORG_MINIMUM_PUBLISHED_TEAMS_SELF_SERVE =
+  process.env.NEXT_PUBLIC_ORG_MINIMUM_PUBLISHED_TEAMS_SELF_SERVE || 2;
```

Comment:
I don't see any reason to create a new environment here; we can achieve a similar result by setting a value directly for `ORG_MINIMUM_PUBLISHED_TEAMS_SELF_SERVE`.

I think the main reason for the issue is setting `ORG_MINIMUM_PUBLISHED_TEAMS_SELF_SERVE` to zero

Reply:
@anikdhabal Ok if you are saying we don't need the env var, then let me remove it.

Anyhow the default value was 2 only. I did add the env var, just for flexibility. For example if someone self-hosts it and wanted to set it to 3 instead, then they can set it using that env var. 👍 

I am frankly not sure if that requirement will arise or not, but still just confirming once again before removing the env var, that should I remove it, whatever you suggest? 🙌

Reply:
Okay keep it. Yeah, it would be beneficial for self-host. Pls add it in .env.example also